src/Game/Object.cpp

```diff
@@ -529,5 +529,52 @@ unsigned short GameObject::lightOrientation() const
     return _lightOrientation;
 }
 
+void GameObject::setLightIntensity(unsigned int intensity)
+{
+    _lightIntensity = intensity;
+}
+
+unsigned int GameObject::lightIntensity() const
+{
+    return _lightIntensity;
+}
+
+void GameObject::setLightRadius(unsigned int radius)
+{
+    _lightRadius = radius;
+}
+
+unsigned int GameObject::lightRadius() const
+{
+    return _lightRadius;
+}
+
+void GameObject::parseFlags(unsigned int flags)
```

Comment:
+1 for *setFlags*
We don't need uint32_t here, cause we performing using binary AND operation on flags value